src/components/AdminPortal/EventRegistrantsModal/Modal/AddOnSpot/AddOnSpotAttendee.tsx

```
@@ -102,18 +104,21 @@ const AddOnSpotAttendee: React.FC<InterfaceAddOnSpotAttendeeProps> = ({
     setIsSubmitting(true);
 
+    const securePassword = generateSecurePassword();
     try {
       const response = await addSignUp({
         variables: {
           ID: orgId,
           name: `${formData.firstName} ${formData.lastName}`.trim(),
           email: formData.email,
-          password: '123456',
+          password: securePassword, // initially it was hardcoded as '123456'
+          signupSource:"ADMIN_ONSPOT"
```

ESLint

[error] Replace `"ADMIN_ONSPOT"` with `·'ADMIN_ONSPOT',`

(prettier/prettier)